README.zh.md

```
@@ -0,0 +1,511 @@
+
+- [借助 AI 辅助开发](#借助-ai-辅助开发)
+- [为什么选择 CrewAI？](#为什么选择-crewai)
+- [快速入门](#快速入门)
+  - [学习资源](#学习资源)
+  - [理解 Flows 与 Crews](#理解-flows-与-crews)
+  - [1. 安装指南](#1-安装指南)
+  - [2. 创建与配置 Crew 项目](#2-创建与配置-crew-项目)
+  - [3. 运行你的 Crew](#3-运行你的-crew)
+- [核心特性](#核心特性)
+- [实战示例](#实战示例)
+  - [快速教程视频](#快速教程视频)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)